File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusRolePermissionVO.kt
package com.sdkwork.backend

data class PlusRolePermissionVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val id: Int? = null,
    val uuid: String? = null,
    val roleId: Int? = null,
    val roleUuid: String? = null,
    val permissionId: Int? = null,
    val permissionUuid: String? = null,
    val enabled: Boolean? = null,
    val description: String? = null,
    val operatorId: Int? = null,
    val status: String? = null
)

File: sdkwork-backend-sdk-csharp/Models/PlusRolePermissionVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusRolePermissionVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? Id { get; set; }
        public string? Uuid { get; set; }
        public int? RoleId { get; set; }
        public string? RoleUuid { get; set; }
        public int? PermissionId { get; set; }
        public string? PermissionUuid { get; set; }
        public bool? Enabled { get; set; }
        public string? Description { get; set; }
        public int? OperatorId { get; set; }
        public string? Status { get; set; }
    }
}
